patches/ember-a11y-testing@5.2.0.patch

```diff
@@ -0,0 +1,9 @@
+diff --git a/test-support/setup-middleware-reporter.d.ts b/test-support/setup-middleware-reporter.d.ts
+index 947f3d5a5e92a4238f115a586bf552b34c97ab10..4360fcea65b4e6a1802ff60fb095c61d572ab413 100644
+--- a/test-support/setup-middleware-reporter.d.ts
++++ b/test-support/setup-middleware-reporter.d.ts
+@@ -1,4 +1,3 @@
+-/// <reference types="ember__test-helpers" />
```

Comment:
Removing this (for now via a patch) was needed due to https://github.com/ember-a11y/ember-a11y-testing/issues/498

Reply:
Nice find!